modules/table/src/test/java/org/apache/ignite/internal/table/distributed/replication/PartitionReplicaListenerTest.java

```diff
@@ -290,6 +314,8 @@ public void beforeTest(@InjectConfiguration DataStorageConfiguration dsCfg) {
 
         lenient().when(safeTimeClock.waitFor(any())).thenReturn(completedFuture(null));
 
+        lenient().when(schemas.waitForSchemasAvailability(any())).thenReturn(completedFuture(null));
```

Comment:
I know that it's not related to this PR, but all these `lenient()` calls can be removed using the `MockitoSettings` annotation

Reply:
I'm not sure it's a good thing to just make all stubbings lenient. I would prefer to be explicit.

Reply:
ok